docs/en/setup/backend/backend-alarm.md

```diff
@@ -152,7 +165,42 @@ Currently, metrics from the **Service**, **Service Instance**, **Endpoint**, **S
 
 Submit an issue or a pull request if you want to support any other scopes in Alarm.
 
-## Webhook
+## Hooks
+Hooks are a way to send alarm messages to the outside world. SkyWalking supports multiple hooks of the same type, each hook can support different configurations. 
+For example, you can configure two Slack hooks, one named `default` and set `is-global: true` means this hook will apply on all `Alarm Rules` **without config** `hooks`.
+Another named `custom1` will only apply on the `Alarm Rules` which **with config** `hooks` and include the name `slack.custom1`.
+
+```yaml
+hooks:
+  slack:
+    default:
+      is-global: true # if true, this hook will apply on all rules, unless a rule has its own specific hook.
```

Comment:
> unless a rule has its own specific hook.

So, if there are `hooks` specified in the rules, the `is-global` is not working.
Am I right? If so, the name `global` seems not proper, it is more like a default rule. We should consider not to set `default` as name in the demo, and change this to `is-default : true`.

Reply:
> So, if there are hooks specified in the rules, the is-global is not working. Am I right?

yes

> If so, the name global seems not proper, it is more like a default rule. We should consider not to set default as name in the demo, and change this to is-default : true

I think `is-global` means you can define more than one global hook in one type. `is-default` mostly means only one?

Reply:
But `global` seems not to be able to be overridden.

Reply:
make sense, changed